src/Raven.Server/Documents/Indexes/IndexStorage.cs

```diff
@@ -1255,6 +1286,8 @@ internal sealed class IndexSchema
             public static readonly Slice ReduceErrorsSlice;
 
             public static readonly Slice LastIndexingTimeSlice;
+            
+            public static readonly Slice ElapsedTimeFromLastQuerySlice;
```

Comment:
Please rename accordingly for consistency. Please see one of above comment

Reply:
Renamed and boxed.